content/en/troubleshooting/faq.md

```diff
@@ -54,3 +54,15 @@ error: failed to transform resource: TOCSS: failed to transform "scss/main.scss"
 We release two set of binaries for technical reasons. The extended version is not what you get by default for some installation methods. On the [release page](https://github.com/gohugoio/hugo/releases), look for archives with `extended` in the name. To build `hugo-extended`, use `go install --tags extended`
 
 To confirm, run `hugo version` and look for the word `extended`.
+
+## What Is The Difference Between Hugo Regular and Hugo Extended and Which Version Should I Choose?
+
+The biggest difference is that Hugo Extended includes [built-in SCSS/SASS support]({{< ref "scss-sass">}}). This comes at the expense of some performance[^1].
```

Comment:
It is the ONLY difference. And the extended part does not come with any performance overhead. I would just drop this perfomance debate. A native Go implementation of SASS would be slightly faster, but a C++ binding version is plenty fast -- and since you don't have anything to comare it with, it's not slower than anything, either.

Reply:
I’ll shorten this section to just the Sass Support.